src/darwin/system-darwin.m

```diff
@@ -105,6 +105,8 @@
     result->_pid = 0;
     frida_image_data_init (&result->_small_icon, 0, 0, 0, "");
     frida_image_data_init (&result->_large_icon, 0, 0, 0, "");
+    if (identifier != nil)
+      [identifier release];
```

Comment:
We could merge the two `release` statements by putting one after the `if/else`. The `nil`-check isn't necessary though, as it's fine to send any message to `nil`.